api/nodes/views.py

```diff
@@ -604,6 +604,11 @@ class NodeRegistrationsList(JSONAPIBaseView, generics.ListCreateAPIView, NodeMix
 
     ordering = ('-modified',)
 
+    def get_serializer_class(self):
+        if self.request.method in ('PUT', 'POST'):
```

Comment:
Should PATCH be included here?

Reply:
I think PATCH is only used for update actions, not creating actions.